src/components/Sections/Header.tsx

```
@@ -30,32 +30,40 @@ const Header: FC = memo(() => {
   );
 });
 
-const DesktopNav: FC<{navSections: SectionId[]; currentSection: SectionId | null}> = memo(
-  ({navSections, currentSection}) => {
+const DesktopNav: FC<{ navSections: SectionId[]; currentSection: SectionId | null }> = memo(
+  ({ navSections, currentSection }) => {
```

ESLint

[error] There should be no space after '{'.

(object-curly-spacing)